lib/mumukit/auth/role.rb

```diff
@@ -28,6 +28,9 @@ def self.parse(role)
       @roles[role] ||= "Mumukit::Auth::Role::#{role.to_s.camelize}".constantize.new(role.to_sym)
     end
 
+    class ExStudent < Role
+      parent :student
```

Comment:
I have just noticed that students will become implicitly ex-students which is kinda awkward. Perhaps what we need to have here is _readers_ or _auditors_ - which is actually something coursera has with the same meaning.